### ISSUE TITLE
Events only fire from switch

### ISSUE THREAD

**Issue by gregpettit** [flatlogic/bootstrap-tabcollapse#23]
In the mobile view of my app, the accordion appears just as it should. This implies to me that the plugin has gone ahead and done the work of changing my tabs to an accordion!

However, if I tap into the show-accordion.bs.tabcollapse event, it is not fired on initial page load into a small viewport. If I am using a browser to simulate mobile (ie. a small viewport), and I expand the window to see tabs and then shrink it back down, the event is indeed now fired!

I don't know if this is a result of the API or a race condition that I can resolve by refactoring. If it's a job for refactoring, I could use some guidance. Right now I have the following inside jQuery's document ready function:

`var myTabs = $("#internalNav");`
`myTabs.on('show-accordion.bs.tabcollapse', function() {`
`  console.log('accordion about to be shown');    `
`});`

When page loads with a small viewport, the line is not logged. When I expand the window and then reduce back to small viewport, it fires.

**Comment by gregpettit** [comment]
My end goal, by the way, is to add chevrons to the tabs. I can't add the chevrons until the bars are rendered. Although I'd still be curious about the original question, is it considered reasonable to supply a new accordionTemplate function that contains my chevrons? 

That approach seems pretty heavy, and not sure it's any better than just modifying the plugin itself. Normally I avoid touching the plugin itself so that I can easily swap in updated versions, and if the updated version has a new accordionTemplate default, my attempt at "future-proofing" fails anyhow.

Thoughts on this approach?

**Comment by julianxhokaxhiu** [comment]
Getting the same bug. I'm trying to debug it to know why it's not triggered by default even if we're already on mobile view.

**Comment by julianxhokaxhiu** [comment]
I think I solved this for my own user case. Practically if you disable `checkStateOnResize` then you can use your own method to "show/hide" the components.

And you can really do it simply using the `.data('bs.tabcollapse')` object inside your item. For example

```js
$myElement
.tabCollapse({
   // ...
   checkStateOnResize: false
   // ...
})
.data('bs.tabcollapse')
.showTabs(); // or .showAccordion()
```

**Comment by markvds** [comment]
I had exactly the same problem. After a little debugging I found out that I called the code in the wrong order. I first created the TabCollapse object and then added the observer. This might seem logical, but the result is that the observer is not in place the first time the event fires.

The solution: first add the observer and then create the TabCollapse object:

```js
$('#tab-id').on('shown-accordion.bs.tabcollapse', function (e) {
    console.log('accordion about to be shown');
});
$('#tab-id').tabCollapse();
